### ISSUE TITLE
MF refactoring: Fix signal transducer + molecular transducer 

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#14268]
A quick (and probably not quite correct) summary of the issue is that these two terms are inconsistent with each other. 

The logical definition of signal transducer includes signaling, which includes ligand activity, which is inconsistent with the definition of ligand activity. This is due to ligand activity being part of signaling. 

A possible fix would be to somehow merge signal transducer + molecular transducer to make sure definitions (logical and textual) include all children. 

@ukemi please fix my description of the issue !! I am not sure I correctly captured our discussion.

Thanks, Pascale

**Comment by dosumis** [comment]
> The logical definition of signal transducer includes signaling, which includes ligand activity

Nope.  It's outside in the ontology, the problem comes when people make part_of links between ligand activity and subclasses of signal transduction in GO-CAM models

> A possible fix would be to somehow merge signal transducer + molecular transducer to make sure definitions (logical and textual) include all children.

Please don't do that.  The intention behind these terms is quite different.  It would be better to just obsolete signal transducer activity (which in its current def is arguably redundant with BP: signal transduction. Or you could make the logical def of signal transducer: 

'molecular transducer' that part_of some 'signal transduction'  - this would exclude ligand activity (not a transducer), but might also be a bit too narrow.

**Comment by pgaudet** [comment]
@thomaspd 
> We still need to fix molecular transducer activity.

**Comment by ValWood** [comment]
What will happen to "molecular transducer activity."?
